src/stores/models/AddressBalance.test.js

```diff
@@ -0,0 +1,54 @@
+import AddressBalance from './AddressBalance';
+import { decimalToSatoshi } from '../../helpers/utility';
+
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn(),
+};
+const navigatorMock = {};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+describe('stores/models/AddressBalance', () => {
```

Comment:
`AddressBalance` or `AddressBalance model` should be sufficient. esp. in case we move the models to a different folder then we would have to change this..